EKF/estimator_interface.cpp

```diff
@@ -478,9 +478,21 @@ void EstimatorInterface::setDragData(const imuSample &imu)
 bool EstimatorInterface::initialise_interface(uint64_t timestamp)
 {
 	// find the maximum time delay the buffers are required to handle
-	float max_time_delay_ms = math::max(_params.baro_delay_ms,
-					math::max(_params.auxvel_delay_ms,
-						_params.airspeed_delay_ms));
+	float max_time_delay_ms = 0.f;
+	// using Barometric altitude
+	if (??) {
+		max_time_delay_ms = math::max(_params.baro_delay_ms, max_time_delay_ms);
+	}
+
+	// using aux vel
+	if (??) {
```

Comment:
The best way here would be to add a en-/disable bit in the EKF2_AID_MASK. Currently, the control logic for the "aux vel" fusion is really crude (no start/stop, health check, ...).

Reply:
@bresch - Done